homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,27 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
+    if (sequence == 1) return 1;
+    if (sequence == 2) return 1;
+    int antepenulminate = 1;
+    int penultimate = 1;
+    int result = 0;
+    if (sequence > 2) {
+        for (int i = 3; i <= sequence; i++) {
+            result = antepenulminate + penultimate;
+            antepenulminate = penultimate;
+            penultimate = result;
+        }
+        return result;
+    }
     return 0;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
+    // edge condition
+    if (sequence == 0) return 0;
+    if (sequence == 1) return 1;
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`